packages/x-components/src/design-system/components/suggestion/default.tokens.scss

```diff
@@ -55,6 +56,12 @@
   --x-number-font-weight-suggestion-default-matching: var(
     --x-number-font-weight-suggestion-default
   );
+  --x-font-family-suggestion-filter-default: var(--x-font-family-suggestion-default);
+  --x-size-font-suggestion-filter-default: var(--x-size-font-suggestion-default);
+  --x-size-line-height-suggestion-filter-default: var(--x-size-line-height-suggestion-default);
+  --x-number-font-weight-suggestion-filter-default: var(
+      --x-number-font-weight-suggestion-default
+  );
```

Comment:
Even if it is not in the figma design, should we add a `text-transform` property? Some of the designs include the whole filter in uppercase.

Reply:
That's a good idea. I agree.